docker/entrypoint.sh

```diff
@@ -9,6 +9,14 @@ set -euo pipefail
 DATABASE_HOST='mysql'
 DATABASE_PORT='3306'
 
+DATABASE_URL=${DATABASE_URL:-mysql://root@127.0.0.1:3306/treeherder}
+SKIP_INGESTION=${SKIP_INGESTION:-False}
+# Skip Data Ingestion when points to other than localhost
+if [ "${DATABASE_URL}" != "mysql://root@localhost:3306/treeherder" ] &&
+   [ "${DATABASE_URL}" != "mysql://root@127.0.0.1:3306/treeherder" ]; then
+    SKIP_INGESTION=True
+fi
```

Comment:
The logic in here is good, however, we need to move it inside of `treeherder/config/settings.py`.

The problem is that this file is *only* used as part of the Docker set up, however, we can set up Treeherder externally in a virtualenv. `settings.py` is a centralized point for both set up.

Reply:
cool. How should I define the `DATABASE_URL` in `settings.py`?

Reply:
Similar idea as the code above but written in Python.